### PR TITLE
Add the page of state of the art

### PR DESCRIPTION
**Pull request by fagomezra**
### Title
Add the page of state of the art
### What does this PR do?
Add the page of state of the art
### Steps to test
1. Run yarn build
2. Run yarn start
3. Go to localhost:3000
4. Go to Monitoring > Deploys the links at .md pages.

#### CheckList
- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

File: docs/docs/research/references.md [hunk 1]
```diff
@@ -11,3 +11,21 @@ custom_edit_url: null
 - [https://leastauthority.com/static/publications/LeastAuthority-Node-Discovery-Protocol-Audit-Report.pdf](https://leastauthority.com/static/publications/LeastAuthority-Node-Discovery-Protocol-Audit-Report.pdf)
 - [https://eprint.iacr.org/2018/236.pdf](https://eprint.iacr.org/2018/236.pdf)
 - [https://ieeexplore.ieee.org/document/9644904](https://ieeexplore.ieee.org/document/9644904)
```
Comment (kuronosec) [inline docs/docs/research/references.md:13]: @fagomezra we should be consistent with formast of the references, with number without numbers?

**Review by kuronosec — CHANGES_REQUESTED** [review]

File: docs/docs/research/references.md (hunk 1 above)
Comment (fagomezra) [reply docs/docs/research/references.md:13]: yes

File: docs/docs/research/stateoftheart.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+id: stateoftheart
+title: State of the Art
+sidebar_label: State of the Art
+description: Tikuna - an Ethereum blockchain network security monitoring system
+keywords: [tikuna, ethereum, research grant]
+hide_table_of_contents: true
+custom_edit_url: null
+---
+
+---
```
Comment (xavier506) [inline docs/docs/research/stateoftheart.md:11]: is this needed ?

File: docs/docs/research/references.md [hunk 3]
```diff
@@ -7,7 +7,21 @@ keywords: [tikuna, ethereum, research grant]
 custom_edit_url: null
 ---
 
-- [https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md](https://github.com/ethereum/devp2p/blob/master/discv5/discv5.md)
-- [https://leastauthority.com/static/publications/LeastAuthority-Node-Discovery-Protocol-Audit-Report.pdf](https://leastauthority.com/static/publications/LeastAuthority-Node-Discovery-Protocol-Audit-Report.pdf)
-- [https://eprint.iacr.org/2018/236.pdf](https://eprint.iacr.org/2018/236.pdf)
-- [https://ieeexplore.ieee.org/document/9644904](https://ieeexplore.ieee.org/document/9644904)
+- [1] https://www.alchemy.com/overviews/ethereum-statistics 
+- [2]https://ethereum.org/669c9e2e2027310b6b3cdce6e1c52962/Ethereum_Whitepaper_-_Buterin_2014.pdf
```
Comment (xavier506) [inline docs/docs/research/references.md:11]:
```
- [1] h
- [2]h
```

some numbers have spacing some don't we should keep them consistent

File: docs/docs/research/stateoftheart.md (hunk 2 above)
Comment (fagomezra) [reply docs/docs/research/stateoftheart.md:11]: Nop, I already deleted them

File: docs/docs/research/references.md (hunk 3 above)
Comment (fagomezra) [reply docs/docs/research/references.md]: I already add the spaces for all lines.

**Review by kuronosec — APPROVED** [review]
LGTM

**Pull request merged**